projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -29,9 +31,19 @@ export class TabParagraphContainerComponent
 
   subscription: Subscription;
 
+  /**
+   * @deprecated since 1.4
+   *
+   * TODO(issue:#5813) Deprecated since 1.4
+   */
+  constructor(
+    componentData: CmsComponentData<CMSTabParagraphContainer>,
+    cmsService: CmsService
+  );
   constructor(
     public componentData: CmsComponentData<CMSTabParagraphContainer>,
-    private cmsService: CmsService
+    private cmsService: CmsService,
+    private activatedRoute?: ActivatedRoute
```

Comment:
Since this is optional, before using it in the component you should do something like:

`if(activatedRoute)` ...